privval/file.go

```diff
@@ -16,7 +16,8 @@ import (
 	cmtos "github.com/cometbft/cometbft/libs/os"
 	"github.com/cometbft/cometbft/libs/protoio"
 	"github.com/cometbft/cometbft/libs/tempfile"
-	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+	cmtproto "github.com/cometbft/cometbft/proto/cometbft/types/v3"
+	cmtproto1 "github.com/cometbft/cometbft/proto/cometbft/types/v1"
```

Comment:
The dispersal of messages between differently versioned packages is annoying and makes the generated packages hard to use. The reason to do it this way was to avoid duplication in message definitions and generated code. But perhaps it's better when all message definitions for an API domain are present in each version (except if they are deprecated), so that e.g. you get everything you need from types by importing `cometbft/types/v3`.